tests/unit/test_cli_performance_edge_cases.py

```
@@ -90,9 +89,14 @@ def large_database_with_mixed_data(self, tmp_path_factory) -> Path:
                             }
                         )
-                    )
 
-        return db_path
+            return db_path
+        finally:
+            # Cleanup backend connection
+            try:
+                await backend.close()
+            except Exception:
+                pass
```

Ruff (0.13.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)